image.c

```diff
@@ -36,6 +36,13 @@
 enum { DEF_GIF_DELAY = 75 };
 #endif
 
+#if HAVE_LIBWEBP
+#include <webp/decode.h>
+#include <webp/demux.h>
+enum { DEF_WEBP_DELAY = 75 };
+Imlib_Image load_webp_firstframe_im(const fileinfo_t *file);
```

Comment:
One of the function is declared here, but the other isn't.

Reply:
This was because `load_webp_firstframe()` is not used outside `image.c`, but `img_load_webp()` is called in `sxiv.c`.

I declared functions only in the scopes in which they were required, but I can understand the confusion. Do you think this should be done differently?

Reply:
Aren't we planning on merging the two functions into one? This shouldn't matter in that case.

Reply:
The way I'm refactoring it, both of these functions will still exist, but I'll create a new function which handles all of the shared code. Then the two existing functions would mostly just be wrappers for the new one with specific single-frame or multi-frame code.

It might be the case that the resulting functions are so trivial that I can then remove one or both of them and fully merge them into one function. Probably I'll leave `img_load_webp()` as a wrapper for a function that takes more arguments and can either load the first frame or all frames of a WebP image into an array of imlib2 images.

Basically, in response to your comment, I'll probably merge the functions into one, but that depends on how nicely the functions behave once I finish extracting all the shared code.